### ISSUE TITLE
errors on compile

### ISSUE THREAD

**Issue by Nackenschelle** [Zz9uk3/ZzukBot_V3#1]
hey there are missing files
i cannot compile this project in visual studio

is there any solution for me?

**Comment by Nackenschelle** [comment]
please respond

**Comment by LordHandLee** [comment]
He won't respond. I have contacted him through other means before and he never helps with any issue. I think he abandoned his project and gave up because he is upset that the source code got leaked and he can't profit.

**Comment by Zz9uk3** [comment]
Uh sorry didnt saw the issue until now. You have to build the different projects in a specific solution. Cant remember in which right now but my guess is that you have to start with the FastCallDll.

> He won't respond. I have contacted him through other means before and he never helps with any issue. I think he abandoned his project and gave up because he is upset that the source code got leaked and he can't profit.

I am just low on time. No need for assumptions.

**Comment by Nackenschelle** [comment]
can you please add a description how to compile this bot?

**Comment by Zz9uk3** [comment]
sorry couldnt compile it myself and had no time to further check on. will investigate why it didnt build for me the next days when i have a minute.

**Comment by Nackenschelle** [comment]
thanks alot
#closed

**Comment by caffeine239** [comment]
Does questing work?

**Comment by Nackenschelle** [comment]
atm nothing is working.
you cannot even compile this bot :(

**Comment by caffeine239** [comment]
This is not true.... i download compiled just foine, even removed the auth server code and made it just compile but not tested. @Nackenschelle give me your email and ill send you that source.

**Comment by Nackenschelle** [comment]
@caffeine239 awesome.
here is my e-mail: andre.harper@web.de

**Comment by Nackenschelle** [comment]
still no e-mail from you -.-

**Comment by caffeine239** [comment]
im sorry data was on my laptop and ive been on my desktop, let me see if i can find it real fast.

**Comment by caffeine239** [comment]
[ZzukBot_WPF.zip](https://github.com/Zz9uk3/ZzukBot_V3/files/2217485/ZzukBot_WPF.zip)

**Comment by Nackenschelle** [comment]
thanks alot

**Comment by LordHandLee** [comment]
@caffeine239 any chance you could upload or send me mmaps? Thanks leeethan186@gmail.com

**Comment by caffeine239** [comment]
> @caffeine239 any chance you could upload or send me mmaps? Thanks [leeethan186@gmail.com](mailto:leeethan186@gmail.com)

https://www.mediafire.com/file/udgpmypeb573yb2/mmaps.rar/file should be the correct mmaps.

**Comment by Zz9uk3** [comment]
I added a readme about how to compile

**Comment by cameronpfs** [comment]
I have read the readme and followed instructions exactly several times spent hours on it not sure what I am missing but when building when I get to ZzukBot_WPF I keep getting:

Severity	Code	Description	Project	File	Line	Suppression State
Error		File 'Resources\favicon.ico' cannot be found.	ZzukBot_WPF			

At first I thought I was just building the wrong thing, however I just can not find a ZzukBot.exe anywhere. Can anyone shed any light?

**Comment by Nackenschelle** [comment]
just replace the missing .ico file with a new icon.

**Comment by cameronpfs** [comment]
I did try that, I then tried ripping it from a lower version which got past
that part, however, it still failed during the build.

**Comment by Nackenschelle** [comment]
latest build works flawless for me

**Comment by NiteKat** [comment]
I was trying to check out this project, but I'm getting an error on building ZzukBot_WPF "error MSB3103: Invalid Resx file. Could not find file 'D:\ZzukBot_V3-master\ZzukBot_WPF\Resources\beep'. Line 222, position 5."
Edit: I managed to get it to compile. I assumed from looking at another project, and the name of this one, that beep should be a sound file. I downloaded a random beep wave, and placed it in the resource folder, removing the file extension. It built, but I don't know if that was actually correct to do.
Now I'm running into an error message on run that says "Download the mmaps first please". Trying to work through that, as the README indicates I should be able to run after building everything.

**Comment by towbes** [comment]
> > @caffeine239 any chance you could upload or send me mmaps? Thanks [leeethan186@gmail.com](mailto:leeethan186@gmail.com)
> 
> https://www.mediafire.com/file/udgpmypeb573yb2/mmaps.rar/file should be the correct mmaps.

Where are these files supposed to go?

**Comment by teakwood54** [comment]
Is this intended to work with modern wow (Classic specifically)? I've added the mmaps (they go in \ZzukBot_V3-master\ZzukBot_V3-master\Release\Internal) and added a filler beep sound file to get it to compile but after it launches wow.exe I get a "A 32 bit processes cannot access modules of a 64 bit process." I'm wondering if that's because this bot was created for the 1.12 client and there have been too many dramatic changes since then.

**Comment by caffeine239** [comment]
> Is this intended to work with modern wow (Classic specifically)? I've added the mmaps (they go in \ZzukBot_V3-master\ZzukBot_V3-master\Release\Internal) and added a filler beep sound file to get it to compile but after it launches wow.exe I get a "A 32 bit processes cannot access modules of a 64 bit process." I'm wondering if that's because this bot was created for the 1.12 client and there have been too many dramatic changes since then.

no this does no work on wow classic 1.13.xx only original from 2004 1.12.1.
